spec/02-integration/05-proxy/05-dns_spec.lua

```diff
@@ -145,5 +145,66 @@ for _, strategy in helpers.each_strategy() do
         assert.response(r).has.status(503)
       end)
     end)
+
+    -- lua-resty-dns is used for DNS query. It will create some UDP sockets
+    -- during initialization. These sockets should be released after Query finish.
+    -- The release is done by explicitly calling a destory method that we patch.
+    -- This test case is to check the UDP sockets are released after the DNS query
+    -- is done.
+    describe("udp sockets", function()
+      local domain_name = "www.example.com"
```

Comment:
Nitpick, feel free to ignore.

```suggestion
      local domain_name = "www.example.test"
```

This way we avoid to resolve to an actual address no matter what changes in `dns_mock`. I know there are a lot of other instances of real hostnames in our tests, but I'm trying to avoid adding more.